scripts/deploy-machine.sh

```diff
@@ -0,0 +1,28 @@
+#!/usr/bin/env bash
+
+set -e #u
```

Comment:
Please use the script preface we started using in our more up-to-date scripts, for consistency reasons.